_data/publications.yaml

```diff
@@ -1,75 +1,88 @@
 ---
-- id: "MerryChristmasHappyFallback-2021-01-12"
-  type: "article"
-  title: "Merry Chistmas and Happy Fallback with Microprofile"
+- url: "https://antoniogoncalves.org/2021/01/12/merry-chistmas-and-happy-fallback-with-microprofile"
   date: "2021-01-12"
+  note: "Build microservices to help Santa schedule his deliveries using Quarkus and\
+    \ MicroProfile."
   author: "Antonio Goncalves"
-  note: "Build microservices to help Santa schedule his deliveries using Quarkus and MicroProfile."
-  url: "https://antoniogoncalves.org/2021/01/12/merry-chistmas-and-happy-fallback-with-microprofile"
+  id: "MerryChristmasHappyFallback-2021-01-12"
+  type: "article"
+  title: "Merry Chistmas and Happy Fallback with Microprofile"
```

Comment:
```suggestion
  title: "Merry Christmas and Happy Fallback with Microprofile"
```

Reply:
but..thats what the title is ... :)

Reply:
LOL, you're right. Just clicked on the link and saw it. /cc @agoncal

Reply:
Oups, even the URL has the typo (`merry-chistmas-`)

Reply:
I'm not touching it. It looks beautiful to me as is :)